Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+pipeline {
+    agent master
+
+    stages {
+        stage('Build') {
+            steps {
+                echo 'cvs checkout..'
+                checkout scm
+            }
+        }
+        stage('Test') {
+            steps {
+                echo 'Testing..'
+                sh 'pip install requirements.txt'
+                sh 'make install'
+                sh 'groovy Jenkinsfile'
```

Comment:
Don't forget to sh 'sh -c \'bash -c \\'python -c \\\\'import sys; sys.exit(0)\\\\'\\'\''

Reply:
that code doesn't work:

```
$ sh 'sh -c 'bash -c \'python -c \\'import sys; sys.exit(0)\\'\'''
sh: sh -c bash: No such file or directory
```